### PR TITLE
Add viewitems, viewkeys, viewvalues implementations for the MultiDict

### PR DESCRIPTION
**Pull request by alexpantyukhin**

### CODE REVIEW

**Comment by alexpantyukhin** [comment]
There is some approach to solve this task. But I need someone reviews this solution)

**Comment by untitaker** [comment]
Just overriding is sensible, but this should be part of the `native_itermethods` decorator.

**Comment by alexpantyukhin** [comment]
Got it, thanks. Fixed.

**Comment by untitaker** [comment]
That's not really what I meant. Instead I thought about changing the *implementation* of the `native_itermethods` decorator such that `viewitems`, `viewkeys` and `viewvalues` will automatically be added without any modifications to the MultiDict.

**Comment by untitaker** [comment]
Could you add some basic tests as well? Thanks!

**Comment by alexpantyukhin** [comment]
Yes, sure. Done. But it's not clear why it fails for the python 2.6.. It seems that the error is not relevant..

**Comment by alexpantyukhin** [comment]
It seems that it's not related with the current task. In different times fails different python versions.

File: werkzeug/datastructures.py [hunk 1]
```diff
@@ -62,7 +66,10 @@ def setmethod(cls, name):
 
     def wrap(cls):
         for name in names:
-            setmethod(cls, name)
+            if PY2:
```
Comment (untitaker) [inline werkzeug/datastructures.py]: IMO the behavior should be exactly as with the builtin dict, that is, provide viewmethods only on Python 2.

**Comment by untitaker** [comment]
Yeah no worries about that.

**Comment by alexpantyukhin** [comment]
Agree. It's done. I have marked as `skip` tests for this functionality on Python 3.

**Comment by untitaker** [comment]
Do you think it would make sense to not create those methods on Python 3? I don't think they *bother* anyone if they're there, but it would be more conforming to `dict` behavior.

**Comment by alexpantyukhin** [comment]
I didn't add these methods for python 3 because the python 3 dictionary implementation doesn't have view methods. Actually you are right that it would not bother anyone. But I tried to be close to the standart dictionary behavior.

**Comment by untitaker** [comment]
Oh sorry, I forgot how this decorator worked.

**Comment by untitaker** [comment]
Thanks!

**Comment by alexpantyukhin** [comment]
You are welcome!

**Pull request merged**